CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# unreleased
+
+* `PrerenderPlugin`:
```

Comment:
I think it's okay not to have a changelog for this change.

Reply:
ok